tensorflow/python/ops/math_ops_test.py

```diff
@@ -171,11 +169,11 @@ def testRounding(self):
     x = np.arange(-5.0, 5.0, .25)
     for dtype in [np.float32, np.double, np.int32]:
       x_np = np.array(x, dtype=dtype)
+      x_tf = constant_op.constant(x_np, shape=x_np.shape)
+      y_tf = math_ops.round(x_tf)
+      y_tf_np = self.evaluate(y_tf)
```

Comment:
This code should be in the `with test_util.device` block (in TensorFlow we place operations on devices rather than Tensors, the device block is making sure that the operations (e.g. constant, round, evaluate) are placed on the GPU when TF executes them.

Overall I think you should just revert all the changes in this file and we should only submit the changes in `sequence_feature_column_test.py`.

Reply:
Thanks @tomhennigan.